### DISCUSSION TITLE
Please update to 1.21

### DISCUSSION THREAD

**Discussion by Cevvity** [iPortalTeam/ImmersivePortalsMod#1615] [Ideas]
Please update to 1.21. I know it is at 1.21.1, but I play on 1.21 as most of my mods are on 1.21, so I would really appreciate it if it was updated to 1.21.

**Comment by Badger505** [comment]
I play on a server that's 1.21 and the portals mod works fine with 1.21, despite being 1.21.1

**Comment by qouteall** [comment]
The version for 1.21.1 should work on 1.21
